### DISCUSSION TITLE
soup path not found

### DISCUSSION THREAD

**Discussion by its-a-me-mario-the-og** [Security-Onion-Solutions/securityonion#7771] [Unsupported Versions]
Hi, I am running a standalone node which has problems with soup. I have tried to do soup to make sure the box is on the latest update and this is the error I got. 
![image](https://user-images.githubusercontent.com/91745779/163015427-0ca6a038-5650-4d19-9056-a45c509d9ab9.png)

It looks like the path it is trying to use doesn't have a file anymore. 
specs:
i5 core processor
8 gbs of RAM
1 tb of SSD
5 tb of HDD

**Comment by dougburks** [comment]
Did soup continue after `Performing upgrade from Security Onion 2.3.61 to Security Onion 2.3.110`?

Are you able to share the full soup log from `/root/soup.log`?

**Reply by its-a-me-mario-the-og** [reply]
No soup did not continue, and here is a copy of the soup log.

[soup.log](https://github.com/Security-Onion-Solutions/securityonion/files/8483543/soup.log)
.

**Reply by dougburks** [reply]
Looks like there are Elasticsearch errors in your soup.log.

Is Elasticsearch running right now?

Have you checked `/opt/so/log/elasticsearch/` for additional clues?

Also, and perhaps related, your 8GB RAM does not meet the minimum requirement of 12GB:
https://docs.securityonion.net/en/2.3/hardware.html#minimum-specs

**Reply by its-a-me-mario-the-og** [reply]
My apologizes the RAM is 12 GBs and Elasticsearch is not running and I did look through for clues and the shards aren't starting. I went ahead and got the log file and redacted some information and attaching it.
[NSM-274-1.log updated.log](https://github.com/Security-Onion-Solutions/securityonion/files/8506670/NSM-274-1.log.updated.log)

**Reply by dougburks** [reply]
The timestamps in that log show 2022-04-07. You'll want to try restarting Elasticsearch and make sure that you're looking at today's log. Also, 12GB is the bare minimum for EVAL. If you're trying to run a true production deployment, you may need more than that.
